option/option.go

```diff
@@ -324,3 +324,15 @@ func (i impersonateServiceAccount) Apply(o *internal.DialSettings) {
 	o.ImpersonationConfig.Delegates = make([]string, len(i.delegates))
 	copy(o.ImpersonationConfig.Delegates, i.delegates)
 }
+
+// WithAttemptDirectPath returns a ClientOption that overrides the default 
+// attempt to use DirectPath
```

Comment:
Could we start this out as an experimental option? See `ImpersonateCredentials` above as an example.

Reply:
Added the comment.